docs/tutorials/user-transpiled-circuits.ipynb

```diff
@@ -31,7 +31,7 @@
        "  <rdf:RDF xmlns:dc=\"http://purl.org/dc/elements/1.1/\" xmlns:cc=\"http://creativecommons.org/ns#\" xmlns:rdf=\"http://www.w3.org/1999/02/22-rdf-syntax-ns#\">\n",
```

Comment:
<p>Thanks for adding "higher is better" here. I think it's still worth to say something like "the value goes from 0 to 1" or "1 means perfect fidelity".</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-ibm-runtime/pull/284/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>